### PR TITLE
Use GraphQL version of Relate-API when available (in Neo4j Desktop)

### PR DESCRIPTION
**Pull request by oskarhane**
Replacement of the reverted https://github.com/neo4j/neo4j-browser/pull/969

Neo4j Browser should behave just as before, but using the GraphQL version rather than the injected version of the relate-api.
We're using the Apollo lib as our client.

The GraphQL queries are loaded on build so we don't need to process AST's on the client side, see https://www.apollographql.com/docs/react/recipes/webpack/.

- [x] No changes when operating standalone
- [x] Automatically connect to running graph in Neo4j Desktop
- [x] Automatically switch to connect to the active graph when user starts a different one in Neo4j Desktop and has Neo4j Browser open
- [x] ~Adapt to color theme preferences set in macOS (not working yet, might not be implemented in Neo4j Desktop, need to check)~ (not implemented on desktop side)
- [x] Update e2e tests (removed them)*

Note: We're still using one function from the injected API and that's `neo4jDesktopApi.getKerberosTicket` since it's not ported to the GraphQL API yet.

* The relate-api will provide a graphql server that can be used for E2E test for graph apps. E2E tests for the desktop env will be added again when that is available.

~Should not be merged until Neo4j Desktop implements the relate-api correctly.~

### CODE REVIEW

**Comment by oskarhane** [comment]
You reviewed the original PR (linked in description) for this @huboneo, could I ask you to have a look at this new one as well?

File: src/browser/components/relate-api/relate-api.hooks.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { useState, useEffect } from 'react'
```
Comment (huboneo) [inline src/browser/components/relate-api/relate-api.hooks.jsx]: no need to use `.jsx` for this file?

File: src/browser/components/relate-api/relate-api.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
+/*
```
Comment (huboneo) [inline src/browser/components/relate-api/relate-api.jsx]: no need for `.jsx`

File: src/browser/modules/DBMSInfo/__snapshots__/MetaItems.test.js.snap [hunk 3]
```diff
@@ -11,7 +11,7 @@ exports[`LabelItems renders empty 1`] = `
       Node Labels
     </h5>
     <div
-      class="Drawer__DrawerSectionBody-sc-5bct1g-5 RMcRs undefined"
+      class="undefined Drawer__DrawerSectionBody-sc-5bct1g-5 RMcRs"
```
Comment (huboneo) [inline src/browser/modules/DBMSInfo/__snapshots__/MetaItems.test.js.snap]: :/

Comment (oskarhane) [reply src/browser/modules/DBMSInfo/__snapshots__/MetaItems.test.js.snap]: ugh, ugly. Fixed.

File: src/browser/components/relate-api/relate-api.hooks.jsx (hunk 1 above)
Comment (oskarhane) [reply src/browser/components/relate-api/relate-api.hooks.jsx]: 👍

File: src/browser/components/relate-api/relate-api.jsx (hunk 2 above)
Comment (oskarhane) [reply src/browser/components/relate-api/relate-api.jsx]: 👍

**Pull request merged**